_layouts/ranking.html

```diff
@@ -0,0 +1,252 @@
+                  <span class="key-table__column__value"><span class="key-table__column__key">Smoothness:</span> ${city.perc_with_smoothness}%</span>
+                  <span class="key-table__column__value"><span class="key-table__column__key">Width:</span> ${city.perc_with_width}%</span>
+                </div>
+              </th>
+              ${city != first ? `
+                <th>
+                  <div class="key-table__column">
+                    <span class="key-table__column__value"><span class="key-table__column__key">Surface:</span> ${first.perc_with_surface}%</span>
+                    <span class="key-table__column__value"><span class="key-table__column__key">Smoothness:</span> ${first.perc_with_smoothness}%</span>
+                    <span class="key-table__column__value"><span class="key-table__column__key">Width:</span> ${first.perc_with_width}%</span>
+                  </div>
+                </th>
+              ` : ''}
+            </tr>
+
+            <tr>
+              <th>
+                <div class="key-table__column">
+                  <span class="title">Barrieren</span>
+                  <span>
+                    (Tag:
+                      <a href="https://wiki.openstreetmap.org/wiki/Tag:barrier%3Dbollard">
+                        barrier=bollard, block, cycle_barrier
+                      </a>
+                    )
+                  </span>
+                  <span>Anzahl: ${city.count_barriers}</span>
+                  <span>Neu Erfasst: ${city.added_barriers}</span>
+                </div>
+              </th>
+              <th>
+                <div class="key-table__column">
+                  <span class="key-table__column__value"><span class="key-table__column__key">Maxwidth:</span> ${city.perc_with_barrier_maxwidth}%</span>
+                </div>
+              </th>
+              ${city != first ? `
+                <th>
+                  <div class="key-table__column">
+                    <span class="key-table__column__value"><span class="key-table__column__key">Maxwidth:</span> ${first.perc_with_barrier_maxwidth}%</span>
+                  </div>
+                </th>
+              ` : ''}
+            </tr>
+            <tr>
+              <th>
+                <div class="key-table__column">
+                  <span class="title">Bordsteine</span>
+                  <span>(Tag: <a href="https://wiki.openstreetmap.org/wiki/Key:kerb">kerbs</a>)</span>
+                  <span>Anzahl: ${city.count_kerbs}</span>
+                  <span>Neu Erfasst: ${city.added_kerbs}</span>
+                </div>
+              </th>
+              <th>
+                <div class="key-table__column">
+                  <span class="key-table__column__value"><span class="key-table__column__key">height measured:</span> ${city.perc_with_kerb_height}%</span>
```

Comment:
otherwise it would also need to be "maxwidth measured" and "width measured"
FYI: the kerb is marked as "with height" if EITHER "flush, lowered, raised" OR the height in meters is given. (we could also change that, but I thought thats makes the most sense)

```suggestion
                  <span class="key-table__column__value"><span class="key-table__column__key">height:</span> ${city.perc_with_kerb_height}%</span>
```